docs/rest_api.md

```diff
@@ -1024,6 +1024,28 @@ Possible errors:
 * 404, if there is no nic attached to `port`
 * 409, if there is already a networking action pending on `port`
 
+#### show_port
+
+`GET /switch/<switch>/port/<port>`
+
+Show the node and nic to which the port is connected.
+
+Response body:
+
+    {
+        u'node': u'mynode',
+        u'nic': u'mynic',
+        u'networks': {u'vlan/1511': u'mynetwork'}
+    }
```

Comment:
This should be displayed as JSON, not a python literal, since this is the HTTP response body. So, get rid of the `u` prefix, and use double quotes.